Web/HTML/html-structure/elements.md

```diff
@@ -11,17 +11,58 @@ category: must-know
 
 tags:
   - introduction
-
+---
+# Elements
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+HTML elements consist of a start and end tag with content in-between.
+Some elements:
+- Paragraph:`<p></p>`
+- Anchors: `<a></a>`
+- Headers: `<h3></h3>`
+- Path `href=""`
+
+Example:
+```
+<h1> 
+  Text in a header element 
+</h1>
+```
+
+It is also very common for HTML elements to be nested. This is common even for the most simple websites.
+
+Example:
+```
+<p>
+  Check out this 
+  <a href="http://www.enkipro.com">
+  website.
+</p>
+```
+
+**Note: Nested elements are very usefull and every website or document has them.**
+
+
+In HTML we also have empty elements or **void** elements. These elements don't have to be closed because they cannot contain content and are also automatically closed by the parser.
+
+The break `<br>` is one of those elements. 
+
+**Note: Even though the closing tag is optional in HTML5, it is best practice to use the closing tag.**
+**This is especially necessary if you want your document to be readable in either XML or XHTML.**
 
 ---
 ## Practice
 
-What two elements are written in this code:
-`<p>Check out this <a href="http://www.enkipro.com">website.</p>`
+
+What element is used in this code:
```

Comment:
I think you'll have to change this one, because if they choose the correct things in the incorrect order, they'll still get it wrong.
Either rephrase the question something like "Which is the enclosing element in this code?" or "what elements are used in this code (in order of appearance)?"